Quintet-iOS/View/ElementView.swift

```diff
@@ -0,0 +1,164 @@
+    @ObservedObject private var coreDataViewModel = CoreDataViewModel()
+    
+    var healthRecords: [RecordMetaData] {
+        return coreDataViewModel.getHealthRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var workRecords: [RecordMetaData] {
+        return coreDataViewModel.getWorkRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var relationshipRecords: [RecordMetaData] {
+        return coreDataViewModel.getRelationshipRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var assetRecords: [RecordMetaData] {
+        return coreDataViewModel.getAssetRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    var familyRecords: [RecordMetaData] {
+        return coreDataViewModel.getFamilyRecords(for: viewModel.selectedYear, month: viewModel.selectedMonth)
+    }
+    
+    func displayRecords(for recordElement: recordElement, records: [RecordMetaData]) -> some View {
+        VStack {
+            Divider()
+                .padding(.horizontal)
+            Button(action: {
+                isShowPopup = true
+            }) {
+                HStack {
+                    viewModel.yearMonthButtonTextRecordVer
+                        .foregroundColor(Color.black)
+                    Image(systemName: isShowPopup ? "chevron.up" : "chevron.down")
+                        .foregroundColor(.black)
+                        .font(.system(size: 15))
+                }
+                .padding(.top, 25)
+                .padding(.trailing, 180)
+            }
+            .sheet(isPresented: $isShowPopup) {
+                CalendarYearMonthPickerPopup(viewModel: viewModel, isShowPopup: $isShowPopup)
+                    .frame(width: 300, height: 400)
+                    .background(BackgroundClearView())
+                    .ignoresSafeArea()
+            }
+            ForEach(records) { metaData in
+                ForEach(metaData.records) { record in
+                    recordCard(icon: record.icon, title: record.title, subtitle: record.subtitle)
+                }
+                .padding(.horizontal, 20)
+            }
+        }
+    }
+    
+    var body: some View {
+        VStack(spacing: 6) {
+            HStack(spacing: 6) {
+                Button(action: {
+                    self.recordIndex = .work
+                }) {
+                    ElementCard(icon: "pencil", title: "일", size: 34, space: 55, fC: recordIndex == .work ? Color.white : Color("DarkGray"), bC: recordIndex == .work ? Color("DarkQ") : Color.white)
+                }
+                
+                Button(action: {
+                    self.recordIndex = .health
+                }) {
+                    ElementCard(icon: "cross.circle.fill", title: "건강", size: 30, space: 30, fC: recordIndex == .health ? Color.white : Color("DarkGray"), bC: recordIndex == .health ? Color("DarkQ") : Color.white)
```

Comment:
```suggestion
                    ElementCard(
                        icon: "cross.circle.fill",
                        title: "건강",
                        size: 30,
                        space: 30,
                        fC: recordIndex == .health ? .white : Color("DarkGray"),
                        bC: recordIndex == .health ? Color("DarkQ") : .white
                    )
```

마찬가지로 위와 같이 수정하는 건 어떨까요?